.github/ISSUE_TEMPLATE/cntt-issue.md

```diff
@@ -1,5 +1,5 @@
 ---
-name: CNTT Issue
+name: Anuket CNTT Issue
```

Comment:
Why are we referring to "Anuket CNTT" as there is no such entity. APplies to many other changes.